client/dashboard/components/site-preview-link/index.tsx

```diff
@@ -26,27 +15,25 @@ type LinkExpiryCopyProps = {
 };
 
 const LinkExpiryCopy = ( { expiresAt }: LinkExpiryCopyProps ) => {
-	const translate = useTranslate();
-
 	const now = moment();
```

Comment:
Should we try using date-fns? The reason I'm mentioning date-fns is because it's currently being used by the [Calendar](https://github.com/Automattic/wp-calypso/tree/trunk/packages/components/src/calendar) component.

Reply:
Not sure about the effort of porting the code to date-fns. I suggest we leave the expiration copy out of this PR for now and create an issue for it.